infra/base-images/base-runner/reproduce

```diff
@@ -30,5 +30,6 @@ fi
 export RUN_FUZZER_MODE="interactive"
 export FUZZING_ENGINE="libfuzzer"
 export SKIP_SEED_CORPUS="1"
+export FUZZ_REPRODUCE_VERBOSE="1"
```

Comment:
This is used by nalloc to be verbose about allocation failures when reproducing (it stays quiet during batch fuzzing)